WelsonJS.Toolkit/WelsonJS.Launcher/Program.cs

```diff
@@ -41,8 +41,22 @@ static Program()
         }
 
         [STAThread]
-        static void Main()
+        static void Main(string[] args)
         {
+            // if set the target file path
+            string targetFilePath = GetTargetFilePath(args);
+            if (!string.IsNullOrEmpty(targetFilePath))
+            {
+                try {
+                    HandleTargetFilePath(targetFilePath);
+                }
+                catch (Exception e)
+                {
+                    _logger.Error($"Initialization failed: {e}");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Avoid string interpolation within log messages, as it can blur the lines between variables and the log content. Instead, opt for structured logging, where variables are passed as separate arguments. This improves the clarity of logs and facilitates efficient log analysis. For more information, refer to https://github.com/NLog/NLog/wiki/How-to-use-structured-logging.